### ISSUE TITLE
Multiple case numbers for 1 client

### ISSUE THREAD

**Issue by siddhesh-athalekar** [CrackDataPots/legallyright#27]
In client details view and 'Add Client' page, user can add multiple case numbers - There can be multiple cases of one client.

**Comment by coderchira** [comment]
That's right. This already exists. for. e.g. But case numbers itself will have separate entry in the DB having mapping to the same client.
From the UI however, user will punch in two entries:
Entry 1 : Client 1 , Case 1, Contact 1
Entry 2 : Client 1 , Case 2 , Contact 1

**Comment by coderchira** [comment]
join to issue #6
